R/validations.R

```diff
@@ -299,18 +305,19 @@ validate_has_variable <- function(data, varname, msg) {
 
 #' Validate that variables has expected number of levels
 #'
-#' @description `r lifecycle::badge("stable")`
+#' `r lifecycle::badge("stable")`
```

Comment:
Won't this push the second paragraph to `@details` section?

```suggestion
#' @description `r lifecycle::badge("stable")`
```